### DISCUSSION TITLE
TPM2 on ASRock's H97M Pro4 Motherboard

### DISCUSSION THREAD

**Discussion by QasimK** [QasimK/qasimk.github.io#20] [Blog Posts]
Comment on [TPM2 on ASRock H97M Pro4 Motherboard](http://qasimk.io/2018/ASRock-Pro4-TPM2/).

**Comment by razvanu** [comment]
will this item work? [TPMS2](https://www.ebay.com/itm/ASRock-TPM2-S-Module-V2-0/382191036061)
with the new [bios, V2.1c](https://mega.nz/#!k1IChbrS!hPs4FcRD441qG0imGsG2r8oYccktyc75yS1XwN5f4D4) for asrock h97 pro4

**Comment by QasimK** [comment]
@razvanu I can't really say. It is labelled as "ASRock TPM2-S Module", and the module that I got was also labelled as "TPM2-S". It sounds like it would work. It's probably worth getting it and returning it if it doesn't work :smiley:

**Comment by prinand** [comment]
Thanks a lot for the BIOS file. I applied it to my H97M Pro4 motherboard (after trying the official 2.30 version first)
but with 2.30 - it did not recognize the TPM module.
after updating the BIOS to your 2.10 version, TPM is now supported
![image](https://user-images.githubusercontent.com/94495282/142082476-d9f3f9c1-c435-4368-a5cd-28f03d4256b6.png)

**Comment by QasimK** [comment]
@prinand I'm glad it worked for you! Thank you for taking the time to let me know :smiley:
